code/game/mecha/equipment/weapons/weapons.dm

```diff
@@ -137,6 +137,7 @@
 		M.stuttering += 20
 		M.ear_deaf += 30
 		M.Knockdown(3)
+		M.Stun(3)
```

Comment:
Remove this one, it already has a chance to stun them just below it and is notoriously bad for stun spamming people